### ISSUE TITLE
testing energy generating cycles reports an error if cofactor is not in the model

### ISSUE THREAD

**Issue by cdanielmachado** [opencobra/memote#220]
I am getting errors for testing production "mql6" and "mql7" because they are not in the model.

According to the paper of Xavier et al [1], these cofactors are not universal. Therefore, this should not be an error. If they are not in the model then maybe the model is just fine.

[1] http://www.sciencedirect.com/science/article/pii/S1096717616302415

**Comment by ChristianLieven** [comment]
Yup, Moritz and I discussed this already. We've implemented the method from [1], which I extended by the menaquinols I found on BiGG. You make a good point that we could look for universal cofactors instead, we could also make these run only if the cofactors can be identified in the model and skip those that we cannot identify. I think we should do both :D Extend our range of tests and skip if the cofactors are unavailable.


[1] Fritzemeier, C. J., Hartleb, D., Szappanos, B., Papp, B., & Lercher,
     M. J. (2017). Erroneous energy-generating cycles in published genome scale
     metabolic networks: Identification and removal. PLoS Computational
     Biology, 13(4), 1–14. http://doi.org/10.1371/journal.pcbi.1005494
